### PR TITLE
Test and Debug PR

### LINKED ISSUE #22: Testing and Debugging required
Tricycle is filled out, but it still needs to be checked for bugs and to make sure it's doing what we want/expect it to do.

### PR DESCRIPTION
**Pull request by dean-krueger**
Changes to FusionPowerPlant which came out of some basic testing and debugging. This PR does not address the lack of formal testing, but rather gets the code doing more or less what we expect it to do. 

This PR Closes #22

### CODE REVIEW

File: src/fusion_power_plant.cc [hunk 1]
```diff
@@ -106,25 +110,35 @@ void FusionPowerPlant::EnterNotify() {
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::Tick() {
-  //pseudocode implementation of Tick():
+
+  // For Debugging:
+  std::cout<<"Timestep "<<context()->time()<<std::endl;
```
Comment (gonuke) [inline src/fusion_power_plant.cc:115]: Even for debugging, it would be good to look at using logging

**Review by gonuke — CHANGES_REQUESTED** [review]
A comment on logging

File: src/fusion_power_plant.cc [hunk 2]
```diff
@@ -227,9 +259,9 @@ void FusionPowerPlant::OperateReactor() {
 }
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::DecayInventories() {
-  //Left empty to quickly check if code builds
-  //tritium_storage.Decay();
-  //tritium_excess.Decay();
+  tritium_storage.Decay();
+  tritium_excess.Decay();
+  sequestered_tritium->Decay(context()->time());
```
Comment (gonuke) [inline src/fusion_power_plant.cc:264]: I think this does the current time by default, just like the resbuf Decay, so you could just call `Decay()` with no arguments here to be like the others.

File: figures/TricycleFPP_Internal_Inv.png
Comment (gonuke) [inline figures/TricycleFPP_Internal_Inv.png:1]: It's nice to see this, but not sure we need it in the repo?  You could include it in the conversation about the PR and also comment on what we should take away from this.

File: src/fusion_power_plant.cc [hunk 3]
```diff
@@ -34,28 +34,32 @@ std::string FusionPowerPlant::str() {
...
   fuel_startup_policy
-      .Init(this, &tritium_storage, std::string("Tritium Storage"),
-            &fuel_tracker)
-      .Set(fuel_incommod, tritium_comp)
-      .Start();
+    .Init(this, &tritium_storage, std::string("Tritium Storage"),
+          &fuel_tracker, std::string("ss"),
+          startup_inventory,
+          startup_inventory)
+    .Set(fuel_incommod)
```
Comment (gonuke) [inline src/fusion_power_plant.cc:57]: is there a reason you no longer include the tritium composition?

File: src/fusion_power_plant.cc [hunk 4]
```diff
@@ -34,28 +34,32 @@ std::string FusionPowerPlant::str() {
...
   fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
     (kDefaultTimeStepDur * 12) * context()->dt());
-  //fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
   blanket_turnover = blanket_size * blanket_turnover_fraction;
-
+  double startup_inventory = reserve_inventory + sequestered_equilibrium; 
+  
   //Create the blanket material for use in the core, no idea if this works...
-  blanket = Material::Create(this, 0.0, 
+   blanket = Material::Create(this, 0.0,
```
Comment (gonuke) [inline src/fusion_power_plant.cc:49]: wrong indentation - may use something like `clang-format` to ensure consistent formatting.

**Review by gonuke — CHANGES_REQUESTED** [review]
This looks pretty much ready to go - just a few nit-picking comments.

File: src/fusion_power_plant.cc (hunk 2 above)
Comment (dean-krueger) [reply src/fusion_power_plant.cc:264]: Unfortunately, it does not. The material version of decay requires a time `Decay (int current_time) {` whereas the ResBuf version requires the same time, but with a default `Decay (int current_time = -1) { `. If you take out the `context()->time()` bit you get an error: `decay cannot use default time with NULL context`.

Comment (gonuke) [reply src/fusion_power_plant.cc:264]: I see... that's because `sequestered_tritium` is no tracked.

**Review by gonuke — APPROVED** [review]
This looks good to move forward.

**Pull request merged**